src/masoniteorm/connections/MySQLConnection.py

```diff
@@ -55,6 +54,16 @@ def make_connection(self):
                 "You must have the 'pymysql' package installed to make a connection to MySQL. Please install it using 'pip install pymysql'"
             )
 
+        try:
+            import pendulum
+            import pymysql.converters
+
+            pymysql.converters.conversions[pendulum.DateTime] = pymysql.converters.escape_datetime
+        except ImportError as e:
+            raise DriverNotFound(
```

Comment:
Should I raise an error here? Or if they don't have pendulum I guess just move on.